src/main/java/org/isf/lab/model/Laboratory.java

```diff
@@ -165,14 +166,22 @@ public void setExam(Exam aExam){
 	public void setLock(int aLock){
 		lock=aLock;
 	}
+	/**
+	 * use <code>getCreatedDate()</code> instead
+	 */
+	@Deprecated
 	public GregorianCalendar getExamDate() {
 		return (GregorianCalendar) examDate;
 	}
+	/**
+	 * the field has been replaced by <code>createdDate()</code> and it's not meant to be managed by the user (Spring managed)
+	 */
+	@Deprecated
 	public void setExamDate(GregorianCalendar exDate) {
 		this.examDate = exDate;
 	}	
 	public void setDate(GregorianCalendar aDate){
-		registrationDate=aDate;
+		labDate=aDate;
```

Comment:
again more formatting here and the next routine

Reply:
Fixed, thanks!